DESCRIPTION

```diff
@@ -36,19 +36,22 @@ Suggests:
     aorsf (>= 0.0.5),
     actuar,
     apcluster,
+    BART (>= 2.9.4),
     C50,
     coin,
     CoxBoost,
     Cubist,
     curl,
     dbarts,
+    dplyr,
     distr6,
     earth,
     flexsurv,
     FNN,
     formattable,
     future,
     gbm,
+    ggplot2,
```

Comment:
we should avoid depending on ggplot2, dplyr and tidyr

Reply:
removed!